src/components/static/live/schedule/Schedules.jsx

```diff
@@ -0,0 +1,64 @@
+"use client";
+import { useEffect, useState } from "react";
+import Title from "../Title";
+import axios from "axios";
+import Event from "./Event";
+const days = ["SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT"];
+const Schedules = () => {
+  const [events, setEvents] = useState([]);
+  const [selectedDay, setSelectedDay] = useState("ALL");
+  const dates = ["ALL", 1, 2, 3, 4, 5, 6, 0];
```

Comment:
pretty sure the numbers can also be done using in built date class functions, only need the all option. also we should only show days that have events

Reply:
the all option shouldnt be used